process/transactionProcessor.go

```diff
@@ -511,16 +590,21 @@ func (tp *TransactionProcessor) getTxFromObserver(observer *data.NodeData, txHas
 	return getTxResponse, true, false
 }
 
-func (tp *TransactionProcessor) getTxFromDestShard(txHash string, dstShardID uint32) (*data.FullTransaction, bool) {
+func (tp *TransactionProcessor) getTxFromDestShard(txHash string, dstShardID uint32, withEvents bool) (*data.FullTransaction, bool) {
 	// cross shard transaction
 	destinationShardObservers, err := tp.proc.GetObservers(dstShardID)
 	if err != nil {
 		return nil, false
 	}
 
+	apiPath := TransactionPath + txHash
+	if withEvents {
+		apiPath += withEventsParam
```

Comment:
Its a module `url` that can be used to compute an URL but code will become more complex. For now I think is more simple this method, because this method is used also in baseProcessor.